src/components/Form/SetAclPermissionForm.jsx

```diff
@@ -50,17 +51,17 @@ const SetAclPermissionForm = () => {
     const permissions = event.target.setAclPerms.value
       ? { read: event.target.setAclPerms.value === 'Give' }
       : undefined;
-    const podUsername = event.target.setAclTo.value;
+    const podToSetPermissionsToURL = event.target.setAclTo.value;
 
-    if (!podUsername) {
-      runNotification('Set permissions failed. Reason: Username not provided.', 5, state, dispatch);
+    if (!podToSetPermissionsToURL) {
```

Comment:
Considering we're going to utilize MUI to help with our validation (making it a required field to input the podUrl of the other user), I think we could remove this `if` statement.

Reply:
> Considering we're going to utilize MUI to help with our validation (making it a required field to input the podUrl of the other user), I think we could remove this `if` statement.

really the only `if` that would need to stay is the check to see if the user put in their own podUrl correct?  All the others will be handled by form validation.

Reply:
That could actually be a form validation as well.

Reply:
We generally use notifications to display server errors to the user. "I tried to submit your form, but the server rejected it because you're not authorized / it's currently down for maintenance / it's a teapot." Errors the client can pick up on can usually be displayed more directly.